interface/modules/custom_modules/oe-module-weno/scripts/weno_log_sync.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ *  @package OpenEMR
+ *  @link    http://www.open-emr.org
+ *  @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @copyright Copyright (c) 2021 Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+use OpenEMR\Common\Crypto\CryptoGen;
+use OpenEMR\Modules\WenoModule\Services\LogProperties;
+use OpenEMR\Modules\WenoModule\Services\WenoPharmaciesJson;
+
+if (isset($_GET['key']) && !empty(isset($_GET['key']))) {
+    start_weno();
+}
+function start_weno()
+{
+    $logsync = new LogProperties();
+    $logsync->logSync();
+}
```

Comment:
Lines 14-21 do not appear to be used anymore, so would remove them. Otherwise, this code can be run without any authentication.

Reply:
I'm not sure on this yet. This was patched out for some reason I need to figure out.

Reply:
@bradymiller Found this is called for the database background_services insert. Also in v6.0.0 sql_upgrade.
I'll figure out a way to handle!